pkg/meta/meta.go

```diff
@@ -1489,6 +1524,79 @@ func (m *Meta) SetSchemaDiff(diff *model.SchemaDiff) error {
 	return errors.Trace(err)
 }
 
+// TableNameKey constructs the key for table name.
+func (*Meta) TableNameKey(dbName string, tableName string) kv.Key {
+	return kv.Key(fmt.Sprintf("%s:%s%s%s", mNames, strings.ToLower(dbName), mNameSep, strings.ToLower(tableName)))
```

Comment:
Could we use `strings.Builder` or direct processing of `[]byte` instead of `fmt.Sprintf` here?